snuba/datasets/storages/transactions_ro.py

```diff
@@ -0,0 +1,27 @@
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.schemas.tables import TableSchema
+from snuba.datasets.storage import ReadableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.transactions import (
+    columns,
+    mandatory_condition_checkers,
+    query_processors,
+    query_splitters,
+)
+
+schema = TableSchema(
+    columns=columns,
+    local_table_name="transactions_local",
+    dist_table_name="transactions_dist_ro",
```

Comment:
Let's keep the same name for now.  `transactions_dist`. So it will work independently on which cluster we put the storage on.